src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -982,6 +982,12 @@ type SynExprAndBang =
         range: range *
         trivia: SynExprAndBangTrivia
 
+    /// Gets the syntax range of this construct
+    member Range: range
+
+    /// Gets the trivia associated with this construct
+    member Trivia: SynExprAndBangTrivia
```

Comment:
Should we include the trivia. Or just take the start range + 4 as `and!` length wont change. So we can avoid having this addition ?

Reply:
IMO we should take it. Correctness is good.